go.mod

```diff
@@ -452,7 +452,7 @@ replace (
 	golang.org/x/sync => golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	golang.org/x/sys => golang.org/x/sys v0.0.0-20210820121016-41cdb8703e55
 	golang.org/x/term => golang.org/x/term v0.0.0-20210615171337-6886f2dfbf5b
-	golang.org/x/text => golang.org/x/text v0.3.6
+	golang.org/x/text => golang.org/x/text v0.3.7
```

Comment:
sorry to keep pressing on this, but there shouldn't be any go.mod changes other than `sigs.k8s.io/apiserver-network-proxy/konnectivity-client`, right?